src/pre-commit.check_working_on_master.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+BRANCH_MASTER="master";
```

Comment:
`;` no needed after variable definition

```suggestion
BRANCH_MASTER="master"
```

Reply:
remove